OverlayPlugin.Core/MemoryProcessors/AtkGui/AtkStageMemory62.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace RainbowMage.OverlayPlugin.MemoryProcessors.AtkStage
+{
+    interface IAtkStageMemory62 : IAtkStageMemory { }
+
+    class AtkStageMemory62 : AtkStageMemory, IAtkStageMemory62
+    {
+        // Offset to global for atkStage singleton instance. ghidra shows `14203d540`, base address of ffxiv_dx11.exe is `140000000`
```

Comment:
I'm surprised that this isn't randomized via ASLR and you can get away without a signature here.  Is it because you added `GetBaseAddress`? I will say I'm a bit suspicious that this will be at all robust across releases.  Signatures allow for a bit of slop unless there's a local change, but a hardcoded address seems fragile.

On top of that, I worry about just reading random pointers with little safety from a hardcoded address.  Will this crash if the address is wrong?

Reply:
It's 100% not robust across releases. I'm still trying to figure out how to get this address via sigscan, there's something I'm not understanding between ghidra decompilation's pointing to this offset via global constants and what it's at during runtime. Ultimately before merge this will need to be addressed and the check found via sigscan.

Reply:
If you update the pointer for 6.28, I'd be happy to try to get you a signature from CheatEngine.

Reply:
The signature I'd expect to work is:
```
E8 ?? ?? ?? ?? 0F BF D5
```
![image](https://user-images.githubusercontent.com/6119598/199592457-eddb613d-dda8-4f31-a749-cdb926b7c9bf.png)
This points to the GetSingleton function:
![image](https://user-images.githubusercontent.com/6119598/199592523-d378e53c-e35a-4d3f-87e6-a3a11cfd12a4.png)
Which should in theory point to the static address of the pointer, in 6.28 that's at `ffxiv_dx11.exe+203c540`, which currently in memory for me is at 11EEACD9CC0.
![image](https://user-images.githubusercontent.com/6119598/199593244-9990b6f1-0d09-4139-873a-9acea6830ad6.png)
The memory at the 2nd screenshot is all 0'd out.
![image](https://user-images.githubusercontent.com/6119598/199593104-35e0f704-0776-4b3b-97b9-802bf99a534f.png)